doctor_visits/delphi_doctor_visits/get_latest_claims_name.py

```diff
@@ -5,7 +5,7 @@
 import datetime
 from pathlib import Path
 
-def get_latest_filename(dir_path, logger):
+def get_latest_filename(dir_path, logger, patch=False):
     """Get the latest filename from the list of downloaded raw files."""
     current_date = datetime.datetime.now()
```

Comment:
do we also need to set the current_date to the issue date to grab the issue date instead here?

Reply:
fixed in [b7f5ecb](https://github.com/cmu-delphi/covidcast-indicators/pull/1977/commits/b7f5ecb69f564c9d3c12832b116a1c53a5400f1a)